packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -42,7 +46,7 @@ export class PluginPathsServiceImpl implements PluginPathsService {
 
         const pluginDirPath = path.join(parentLogsDir, this.gererateTimeFolderName(), 'host');
         await this.fileSystem.createFolder(pluginDirPath);
-
+        await this.cleanupOldLogs(parentLogsDir);
```

Comment:
How often is this method called? Would it slow it down?

Reply:
It gets called twice for some reason, the remove of each folder is intentionally not "awaited" to. however each call does scan the sub-folder names of the root log folder, so that could add some time.

Note that it is strange that it is called twice, because if two calls are done in **different seconds** then two log session folders could be created with timestamp different by 1 second.

Reply:
we should never wait on cleanup

Reply:
`this.cleanupOldLogs(parentLogsDir)` is now called without `await`
To avoid waiting on cleanup.